# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/iproute2/hotfix-path-max.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

iplink.c: In function 'iplink_parse':
iplink.c:663:43: error: 'PATH_MAX' undeclared (first use in this function); did you mean 'AF_MAX'?
  663 |                                 char path[PATH_MAX];

--- iproute2-6.12.0/ip/iplink.c.vanilla	2024-12-05 20:50:04.769772978 +0100
+++ iproute2-6.12.0/ip/iplink.c	2024-12-05 20:50:17.969772892 +0100
@@ -11,6 +11,7 @@
 #include <fcntl.h>
 #include <dlfcn.h>
 #include <errno.h>
+#include <limits.h>
 #include <sys/socket.h>
 #include <linux/if.h>
 #include <linux/if_packet.h>
